library/src/containers/Schemas/SchemaExample.tsx

```diff
@@ -8,33 +8,35 @@ import { SCHEMA_EXAMPLE_TEXT } from '../../constants';
 
 interface Props {
   title?: string;
-  schema: Schema;
+  schema?: Schema;
+  example?: object;
 }
 
 export const SchemaExampleComponent: React.FunctionComponent<Props> = ({
   title,
   schema,
+  example,
 }) => {
-  const example = JSON.stringify(
-    schema.example ? schema.example : generateExampleSchema(schema),
-    null,
-    2,
-  );
+  const schemaExample =
+    schema && schema.example
+      ? schema.example
+      : schema && generateExampleSchema(schema);
+  const exampleString = JSON.stringify(example || schemaExample, null, 2);
```

Comment:
Please change it to `JSON.stringify(example || schemaExample || "", null, 2)` because if schema and example props don't exist then pass empty string to JSON.stringify. I know that stringify works also with null or undefined, but this is only for safety.

Reply:
yeap, you are right, I changed that schema is also optional now